Assets/ScriptableObjects/PlayerMovement.asset

```diff
@@ -17,8 +17,12 @@ MonoBehaviour:
   maxSpeed: 10
   maxAcceleration: 100
   maxAirAcceleration: 100
+  defaultGravityScale: 10
```

Comment:
Out of curiosity why is the default `10` here? I would have thought `1` would be the default gravity scale

Reply:
Good question. I also wondered about that, but it basically doesn't matter because if the player is in the air at all the jump script takes over and sets the gravity scale based on whether the player is going up or down. The default is only active when the player is grounded, so I just set it to be the same as the falling gravity scale